my-dsl.md

```diff
@@ -1,38 +1,30 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+[Protocol Buffers](https://developers.google.com/protocol-buffers/)
 
 
 # Domain
-_Describe the language's domain in five words._
+Sending and processing language-neutral data.
 
 
 # Computational model
-_What is the underlying computational model of this language? To answer this 
-question, provide a high-level description (no more than 100 words) of the 
-computation that occurs when someone executes a program in this language._
+The file that defines the messages is compiled, and then instances of the messages used by other applications are encoded to binary based on the object file.
```

Comment:
Isn't the DSL also how these are being used in other languages? They provide compilers for it in other languages, so I think that should be part of the computational model.